### DISCUSSION TITLE
2023-10-18

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#673] [daily_meeting]
### 데일리 미팅

개인일정
- 노아 5시

### 오늘의 할 일

- 무중단 배포
- 서비스 페이지 움짤 캡처

### 내일의 할 일

**Comment by jaehee329** [comment]
### 체크인
7점 - 밀린 미션 해야겠다

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 편하게 잠 좀 자고 싶네용... 미션 오늘 좀 끝내봅시다!

**Comment by aak2075** [comment]
### 체크인

8점 - 이번주 너무 힘드네요

**Comment by woo-jk** [comment]
### 체크인
8점 - 알고리즘 공부하자

**Comment by woosung1223** [comment]
### 체크인
7점 - 너무 졸립니다

**Comment by yeopto** [comment]
### 체크인

5점 - 너무졸려요

### 체크아웃

7점 - 어드민 끝이 보입니다.
